### PR TITLE
call_permission: relax call_permission validation

### PR DESCRIPTION
**Pull request by pc-m**
this field is for display only there's no need for anything complex here

### CODE REVIEW

**Review by fblackburn1 — CHANGES_REQUESTED** [review]
Do you know why/history of unique constraint?
user import is broken, you need to find_by(tenant&name) and get_by(tenant&name)

File: wazo_confd/plugins/call_permission/validator.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+        metadata = {'tenant_uuid': tenant_uuid, 'name': name}
+        raise errors.resource_exists('CallPermission', **metadata)
+
+
+def build_validator():
+    return ValidationGroup(
+        create=[UniqueNameField(call_permission_dao)],
+        edit=[Optional('name', UniqueNameField(call_permission_dao, edit=True))],
+    )
```
Comment (fblackburn1) [inline wazo_confd/plugins/call_permission/validator.py]: Helpers already exist to use tenant
https://github.com/wazo-platform/wazo-confd/blob/master/wazo_confd/plugins/skill/validator.py

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 21.10
+
+* The name of call_filters does not have to be unique anymore
+
+
```
Comment (fblackburn1) [inline CHANGELOG.md:7]: Must be updated to be unique by tenant

**Comment by pc-m** [comment]
recheck

**Review by fblackburn1 — APPROVED** [review]

**Pull request merged**